index.html

```diff
@@ -53,6 +48,171 @@ <h2>I am a Software Developer</h2>
                     </ul>
                 </div>
             </section>
+             <!-- =============
+            Recent Works section
+        =========== -->
+        <section class="recent-works">
+            <h3>My Recent Works</h3>
+            <hr>
+            <div class="recent-work">
+                <a href=""><img src="./image/Img_Placeholder_1.png" alt="Women doing yoga"></a>
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tech-used">
+                    <li>CSS</li>
+                    <li>html</li>
+                    <li>bootsrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="project-button " type="button">See Project</button>
+            </div>
+            <div class="recent-work">
+                <a href="#"><img src="./image/Placeholder12.png" alt="Women doing yoga"></a>
+
+                <button class="project-button " type="button">See Project</button>
+            </div>
+            <div class="cards-section">
+                <div class="work-card">
+                    <div class="card-availability">
+                        <h4>Data Dashboard Healthcare</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
```

Comment:
Excellent work using an unordered list to show the technologies of the project 😎